README.md

```diff
@@ -1,6 +1,6 @@
-echo "# codereviewtest" >> README.md
-git init
-git add README.md
```

Comment:
I don't think this is correct

Reply:
interesting...

Reply:
this is another test.. adding a comment would this start a new review?